src/synchronizeContacts.js

```diff
@@ -14,6 +15,7 @@ const haveRemoteFieldsChanged = (
     'name.givenName',
     'cozy.0.url',
     'jobTitle',
+    'trashed',
```

Comment:
I guess that `trashed` will always be false for remote contacts? maybe add a comment about this?